lib/test-worker.js

```diff
@@ -86,6 +86,12 @@ var oldNodeModulesPaths = module.constructor._nodeModulePaths;
 module.constructor._nodeModulePaths = function () {
 	var ret = oldNodeModulesPaths.apply(this, arguments);
 	ret.push(nodeModulesDir);
+	if (process.env.NODE_PATH) {
+		var osSplitChar = /^win/.test(process.platform) ? ';' : ',';
```

Comment:
Perhaps use [`is-windows`](https://www.npmjs.com/package/is-windows). A separate package is likely preferable over inline regex fu.

---

> If the NODE_PATH environment variable is set to a colon-delimited list of absolute paths, then Node.js will search those paths for modules if they are not found elsewhere. (Note: On Windows, NODE_PATH is delimited by semicolons instead of colons.)

[(source)](https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders)

The non-Windows separators are colons, not commas.